lib/data/entity.js

```diff
@@ -124,5 +127,56 @@ const streamEntityCsvs = (inStream, properties) => {
   return PartialPipe.of(inStream, rootStream, csv());
 };
 
+// Paging is done at the database level
+const streamEntityOdata = (inStream, properties, domain, originalUrl, query, tableCount) => {
+  const serviceRoot = getServiceRoot(originalUrl);
+  const { limit, offset, shouldCount } = extractPaging(query);
+
+  const schema = {
+    name: null,
+    label: null
+  };
+
+  for (const property of properties) {
+    schema[property.name] = null;
+  }
```

Comment:
I don't think it's a huge deal here, but since the keys here are user-defined, I think it'd be better to create the object with `Object.create(null)`.